src/modules/watch/service.test.js

```diff
@@ -8,24 +11,91 @@ describe('test suite for WatchService', () => {
   const mongol = new Mongol(MONGO_URL, 'testdb')
   let db
   let service
+  let watch
+  const objectID = new ObjectID()
 
   beforeAll(async () => {
     await mongol.connect()
     db = mongol.database
   })
 
   beforeEach(async () => {
+    const mockWatch = {
```

Comment:
```suggestion
    const mockWatch1 = {
```
Đã đánh số thì nên đánh số tất chứ, cái bên dưới trở thành `mockWatch2`.